test/integration/targets/win_iis_website/tasks/main.yml

```diff
@@ -0,0 +1,72 @@
+---
+# Cannot use win_feature to install IIS on Server 2008.
+# Run a brief check and skip hosts that don't support
+# that operation
+- name: check if win_feature will work on test host
+  win_command: powershell.exe "Get-WindowsFeature"
+  register: module_available
+  failed_when: False
+
+# Run actual tests
+- block:
+  # Setup for tests
+  - name: this is too finicky reboot before feature install
+    win_reboot:
+
+  - name: ensure IIS features are installed
+    win_feature:
+      name: Web-Server
+      state: present
+      includ_sub_features: True
```

Comment:
Well if this is working without this actually there I don't see a reason why we need all sub features